dronekit/__init__.py

```diff
@@ -1022,7 +1013,7 @@ def __init__(self, handler):
 
         # Cache all updated attributes for wait_ready.
         # By default, we presume all "commands" are loaded.
-        self._ready_attrs = set(['commands'])
+        self._ready_attrs = {'commands'}
```

Comment:
This I don't understand.  Moving from a set to a dictionary is strange.

Reply:
It’s not a dict. It’s a set literal. https://stackoverflow.com/questions/36674083/why-is-it-possible-to-replace-sometimes-set-with